boards/beagle/beagleplay_cc1352/board_antenna.c

```diff
@@ -0,0 +1,107 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2021 Florin Stancu
+ * Copyright (c) 2021 Jason Kridner, BeagleBoard.org Foundation
+ */
+
+/*
+ * Implements the RF driver callback to configure the on-board antenna
+ * switch.
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+
+#include <ti/drivers/rf/RF.h>
+#include <driverlib/gpio.h>
+#include <driverlib/ioc.h>
+#include <driverlib/rom.h>
+
+/* DIOs for RF antenna paths */
+#define BOARD_RF_HIGH_PA 29 /* TODO: pull from DT */
+#define BOARD_RF_SUB1GHZ 30 /* TODO: pull from DT */
```

Comment:
@gmarull is it blocking that these are not in dts?

Reply:
Well, this is the exact same file as [beagleconnect_freedom/board_antenna.c](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/beagle/beagleconnect_freedom/board_antenna.c).

Reply:
I guess it was acceptable to allow this once, but if no proper maintainership is offered (ie, do the to-dos), then -1 on introducing this again.

Reply:
@Ayush1325 - you can probably just pull in the source from the other directory using a relative path in cmake.

However, I would also like to see an issue (enhancement? Bug maybe?) filed and for @gmarrull's concerns to be addressed from the original source.

@gmarrull - please correct me if I'm wrong, but I think some constants need to be declared in a header file under `include/zephyr/dt-bindings` and for those constants to be incorporated into the board `.dts` or perhaps a parent `.dtsi`. Likely the `.c` file which handles this needs to be cleaned up.

It might help to be very specific about change requests because it might not be obvious what needs to be fixed. An issue would be a good place to be very specific about those things and it can be assigned the a board's owner.